tests/test_fixtures/rule_based_profiler/example_notebooks/MultiBatchExample_ConfiguredAssetSQLExample_SQL.ipynb

```diff
@@ -85,58 +85,23 @@
-    "\n",
-    "It gives the following output, which corresponds to our two tables: \n",
-    "\n",
-    "```bash \n",
-    "Data Connectors:\n",
-    "    whole_table : ConfiguredAssetSqlDataConnector\n",
-    "    Available data_asset_names (1 of 1):\n",
-    "        yellow_tripdata_sample_2020 (1 of 1): [{}]\n",
-    "```"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "6a7a8284",
-   "metadata": {},
-   "source": [
+    "The DataConnector contains 2 `assets`, both associated with the `table_name` named`yellow_tripdata_sample_2020`.\n",
     "\n",
-    "The partitioner `by_pickup_month` is configured by us, and uses a `splitter_method` to split the table values into multiple batches. The splitter we use is `split_on_year_and_month`, which creates Batches according to the `pickup_datetime` column which of type `timestamp` in the database schema. \n",
+    "The asset `yellow_tripdata_sample_2020_full` contains no other parameter other than the `table_name` and optional `schema_name`, which mean the whole table will be loaded as one Batch in the asset. \n",
     "\n",
-    "We also include  `whole_table` partitioner, which is built-in into the `SimpleSqlalchemyDatasource`.  With the `whole_table` partitioner, we are able to refer to the `yellow_tripdata_sample_2020` table as a data asset with a single batch. \n",
-    "    \n",
-    "Here is the output, which shows the data asset `yellow_tripdata_sample_2020` with 12 batches, each associated with a different `pickup_datetime`. These become our `batch_identifiers` that distinguish one `Batch` from another.\n",
+    "The asset `yellow_tripdata_sample_2020_by_year_and_month` contains `table_name` and `schema_name`, as well as splitter configuration. The splitter we use is `split_on_year_and_month`, which creates Batches according to the `pickup_datetime` column which of type timestamp in the database schema.\n",
     "\n",
-    "```bash\n",
-    "Data Connectors:\n",
-    "\tby_pickup_month : ConfiguredAssetSqlDataConnector\n",
-    "        Available data_asset_names (1 of 1):\n",
-    "            yellow_tripdata_sample_2020 (3 of 12): [{'pickup_datetime': {'year': 2020, 'month': 1}}, {'pickup_datetime': {'year': 2020, 'month': 10}}, {'pickup_datetime': {'year': 2020, 'month': 11}}]\n",
-    "            \n",
-    "    whole_table : ConfiguredAssetSqlDataConnector\n",
-    "        Available data_asset_names (1 of 1):\n",
-    "            yellow_tripdata_sample_2020 (1 of 1): [{}]\n",
-    "```"
+    "**Note**: This example only uses `splitters` but sampling can also be used. For more information, please refer to the document [How to configure a DataConnector for splitting and sampling tables in SQL](https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/advanced/how_to_configure_a_dataconnector_for_splitting_and_sampling_tables_in_sql)"
    ]
   },
   {
```

Comment:
This notebook is un-executed, can we commit an executed version so that people can view the output e.g. on github without having to run it?